lib/full_text_extractor.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "net/http"
+require "uri"
+##
+# This class sends a PDF IO object to the Solr text extraction service
+# and retrieves the JSON that is returned by Solr
+class FullTextExtractor
```

Comment:
If this only has one method and no real state, it may as well be a module.

Reply:
It could always have more 😄 

I like doing even small classes like this because the resulting specs look nicer to me. If you feel *super strongly* about this I'll change it though.

Reply:
I feel strongly, but not super strongly.